Cargo.toml

```diff
@@ -19,7 +19,7 @@ endpoint-sec-sys = { version = "0.2.0", path = "./endpoint-sec-sys" }
 block2 = "0.2.0"
 libc = { version = "0.2", features = ["extra_traits"] }
 mach2 = "0.4"
-objc2 = "0.4.0"
```

Comment:
Hmm, I'm realizing that it's bumping the deps in the cargo.toml, instead of only bumping the cargo.lock when that's enough. I wonder if we can do something about that? This is kinda annoying because it forces our downstreams to have the absolute latest and greatest of their own deps (which they may not want to do).

Reply:
Well that sucks. Looks like what we want is to set `versioning-strategy` to `increase-if-necessary` (see [this](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#versioning-strategy), but that isn't implemented for cargo. We can set it to `lockfile-only`, but then we won't get dependabot MRs for major dep updates.

Reply:
maybe renovate would be better here ?

Reply:
I checked, and renovate doesn't fare much better (though it's potentially fixable - see https://github.com/renovatebot/renovate/issues/22820)

Reply:
Here the problem is that we specified patch versions, being less precise should work

Reply:
Did https://github.com/HarfangLab/endpoint-sec/pull/8